### DISCUSSION TITLE
全屏播放时帧数上千音画不同步

### DISCUSSION THREAD

**Discussion by da4411** [hooke007/mpv_PlayKit#450] [Q&A 问答]
![屏幕截图(7)](https://github.com/hooke007/MPV_lazy/assets/128574512/83760fb3-f4f4-476f-bcd0-5ed68ac15471)

如图按出菜单后帧数就下降到120了，但是瞬间帧数变化很大导致很卡。
显卡是5700xt已经更新最新驱动了，有大佬知道怎么解决这个问题吗

![屏幕截图(8)](https://github.com/hooke007/MPV_lazy/assets/128574512/01b1d2a8-1971-4df3-9942-5404142caca7)

**Comment by hooke007** [comment]
我建议你看看faq
